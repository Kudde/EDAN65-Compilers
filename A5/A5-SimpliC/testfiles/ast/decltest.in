int a() {
        int x;
        int y;
        x = 5;
        y = x;
        int k = 3;
}
