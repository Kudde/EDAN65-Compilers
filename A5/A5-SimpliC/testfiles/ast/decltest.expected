Program
  List
    Fun
      IntType ID="int"
      IdDecl ID="a"
      Param
        List
      Block
        List
          DeclStmt
            IntType ID="int"
            IdDecl ID="x"
            Opt
          DeclStmt
            IntType ID="int"
            IdDecl ID="y"
            Opt
          AssignStmt
            IdUse ID="x"
            Literal NUMERAL="5"
          AssignStmt
            IdUse ID="y"
            IdUse ID="x"
          DeclStmt
            IntType ID="int"
            IdDecl ID="k"
            Opt
              Literal NUMERAL="3"
